traefik.go

```diff
@@ -247,21 +268,32 @@ func prepareServer(router *mux.Router, globalConfiguration *GlobalConfiguration,
 	negroni.UseHandler(router)
 
 	if oldServer == nil {
+		tlsConfig, err := createTLSConfig(globalConfiguration.CertFile, globalConfiguration.KeyFile)
```

Comment:
You could bring that up (before `if oldServer ==nil`), as you do exactly the same it in the `if` and in the `else`.